### DISCUSSION TITLE
Font Size Removed from Heading Widget

### DISCUSSION THREAD

**Discussion by projectpi-ca** [elementor/elementor#25371] [Developers Community]
### Start a discussion

I reached out to Elementor Support and they told me to come here. I've created a feature request but it was ignored, so I'm posting here to see if anyone has any insights.

I managed over 20 sites that use Elementor. After updating to the latest version, the heading widget no longer shows the font size option.

After looking into the code, I found the following lines in this file: elementor/includes/widgets/heading.php

```
		$this->add_control(
			'size',
			[
				'label' => esc_html__( 'Size', 'elementor' ),
				'type' => Controls_Manager::SELECT,
				'options' => [
					'default' => esc_html__( 'Default', 'elementor' ),
					'small' => esc_html__( 'Small', 'elementor' ),
					'medium' => esc_html__( 'Medium', 'elementor' ),
					'large' => esc_html__( 'Large', 'elementor' ),
					'xl' => esc_html__( 'XL', 'elementor' ),
					'xxl' => esc_html__( 'XXL', 'elementor' ),
				],
				'default' => 'default',
				'condition' => [
					'size!' => 'default', // a workaround to hide the control, unless it's in use (not default).
				],
			]
		);`
```

Does anyone know the reason for this, and what Elementor's plan is for this option?

Feature Request: https://github.com/orgs/elementor/discussions/25174

**Comment by nicholaszein** [comment]
Hi @projectpi-ca!

Please refer to this comment!

https://github.com/orgs/elementor/discussions/25174#discussioncomment-11353705

Kind regards
